TestcsFastFloat/Basic/BasicTests.cs

```diff
@@ -307,6 +307,10 @@ private void TestInfinity_Double(string sut, double expected_value)
     [InlineData("0.00000000000000000000000000000000000002350988561514728583455765982071533026645717985517980855365926236850006129930346077117064851336181163787841796875", 0.00000000000000000000000000000000000002350988561514728583455765982071533026645717985517980855365926236850006129930346077117064851336181163787841796875f)]
     [InlineData("0.00000000000000000000000000000000000001175494210692441075487029444849287348827052428745893333857174530571588870475618904265502351336181163787841796875", 0.00000000000000000000000000000000000001175494210692441075487029444849287348827052428745893333857174530571588870475618904265502351336181163787841796875f)]
     [Theory]
-    private void TestGeneral_Float(string sut, float expected_value) => Assert.Equal(expected_value, FastFloatParser.ParseFloat(sut));
+    private void TestGeneral_Float(string sut, float expected_value)
+    {
+      Assert.Equal(expected_value, FastFloatParser.ParseFloat(sut));
+      Assert.Equal(expected_value, FastFloatParser.ParseFloat(sut.AsSpan()));
```

Comment:
The ReadOnlySpan<char> path is almost the same as for string, so I think just these tests are enough.